src/test/server/store-test.js

```diff
@@ -0,0 +1,118 @@
+        it('Should return a list of users', () => {
+            assert(users, UserStore.getUsers());
+        });
+    });
+
+    describe('Actions', () => {
+        describe('addNewUser', () => {
+            it('A new user should be added', () => {
+                const user = new UserInfo(
+                    {
+                        id:      11,
+                        name:    'Vika',
+                        phone:   '89500418181',
+                        email:   'vika.chernookaya@mail.ru',
+                        website: 'github.com'
+                    }
+                );
+                users.push(user);
+                UserAction.addNewUser(user);
+                assert(users, UserStore.getUsers());
+            });
+            it('User with incorrect fields should not be added', () => {
+                const user = new UserInfo(
+                    {
+                        id:      11,
+                        name:    'Vika',
+                        phone:   'phone0418181',
+                        email:   'vika.chernookaya@mail.ru',
+                        website: ''
+                    }
+                );
+                UserAction.addNewUser(user);
+                assert(users, UserStore.getUsers());
+            });
+            it('Existing user should not be added', () => {
+                const user = new UserInfo(
+                    {
+                        id:      11,
+                        name:    'Ervin Howell',
+                        phone:   '0418181',
+                        email:   'ervin@mail.ru',
+                        website: 'github.com'
+                    }
+                );
+                UserAction.addNewUser(user);
+                assert(users, UserStore.getUsers());
+            });
+            it('Existing user should not be added', () => {
+                const user = new UserInfo(
+                    {
+                        id:      11,
+                        name:    'Ervin Howell',
+                        phone:   '111111',
+                        email:   'ervin@mail.ru',
+                        website: 'github.com'
+                    }
+                );
+                users.push(user);
+                UserAction.addNewUser(user, true);
+                assert(users, UserStore.getUsers());
+            })
+        });
+        describe('updateUser', () => {
+            it('User should be updated', () => {
+                const user = new UserInfo(
+                    {
+                        id:      2,
+                        name:    'Ervin',
+                        phone:   '555',
+                        email:   'ervin@mail.ru',
+                        website: 'github.com'
+                    }
+                );
+                users[1] = user;
+                UserAction.updateUser(user);
+                assert(users, UserStore.getUsers());
+            });
+            it('User should not be updated', () => {
```

Comment:
we should name tests from the following point of view:
- what we are going to do
- what we exactly expect